electron/src/components/privilege-checker/privilege-checker.component.ts

```diff
@@ -50,14 +63,19 @@ export class PrivilegeCheckerComponent {
             case 'linux':
                 permissionSetter = this.setUpPermissionsOnLinux();
                 break;
+            case 'win32':
```

Comment:
Please comment out this branch, and add a comment explaining that hidapi shouldn't need privilege escalation on Windows.